src/atomate2/vasp/flows/eos/base.py

```diff
@@ -0,0 +1,207 @@
+    """
+    Workflow to generate initial double relaxation for EOS fitting.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker1 : .BaseVaspMaker
+        Maker to use to generate the first relaxation.
+    relax_maker2 : .BaseVaspMaker
+        Maker to use to generate the second relaxation.
+
+    NB: WAVECARs are copied over to speed up calculation
+    """
+
+    name: str = "EOS double relax"
+    relax_maker1: BaseVaspMaker | None = field(default_factory=EosRelaxMaker)
+    relax_maker2: BaseVaspMaker = field(
+        default_factory=lambda: EosRelaxMaker(
+            copy_vasp_kwargs={"additional_vasp_files": ("WAVECAR",)}
+        )
+    )
+
+
+@dataclass
+class EosMaker(Maker):
+    """
+    Workflow to generate energy vs. volume data for EOS fitting.
+
+    First relax a structure using relax_maker.
+    Then perform a series of deformations to the relaxed structure, and
+    evaluate single-point energies with static_maker.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker : .BaseVaspMaker
+        Maker to generate the relaxation, defaults to DoubleRelaxMaker
+    deformation_maker : .BaseVaspMaker
+        Maker to generate deformations + single-points, defaults to TransmuterMaker
+    static_maker : .BaseVaspMaker
+        Optional Maker to generate statics from transmutation.
+        Original atomate workflow did not include statics, including it here
+    strain : tuple[float]
+        Percentage linear strain to apply as a deformation, default = -5% to 5%
+    number_of_frames : int
+        Number of strain calculations to do for EOS fit, default = 6
+    postprocessor : .job
+        optional postprocessing step
+    """
+
+    name: str = "EOS Maker"
+    relax_maker: Maker = field(default_factory=EosDoubleRelaxMaker)
+    deformation_maker: Maker = field(
+        default_factory=lambda: DeformationMaker(
+            copy_vasp_kwargs={"additional_vasp_files": ("WAVECAR",)}
+        )
+    )
+    static_maker: Maker | None = None
+    linear_strain: tuple[float, float] = (-0.05, 0.05)
+    number_of_frames: int = 6
+    postprocessor: Job = postprocess_EOS
+
+    def make(self, structure: Structure, prev_dir: str | Path | None = None):
+        """
+        Run an EOS VASP job.
+
+        Parameters
+        ----------
+        structure : Structure
+            A pymatgen structure object.
+        prev_dir : str or Path or None
+            A previous VASP calculation directory to copy output files from.
+        """
+        jobs: dict[str, list[Job]] = {"relax": [], "static": [], "postprocess": []}
+
+        relax_flow = self.relax_maker.make(structure=structure, prev_dir=prev_dir)
+        relax_flow.name = "EOS equilibrium relaxation"
+        equil_props = {
+            "relax": {
+                "E0": relax_flow.output.calcs_reversed[0].output.energy,
```

Comment:
If you use `equil_static.output.output.energy` this should be less VASP specific. See below for structure and other instances where you access `calcs_reversed`.